torch_ttnn/passes/constant_folding_pass.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from torch._subclasses.fake_tensor import unset_fake_temporarily
+from torch.fx.passes.infra.pass_base import PassBase, PassResult
+
+
+class ConstantFoldingPass(PassBase):
```

Comment:
This is a part of the other change. Is that expected?

Reply:
Because this branch is base on [dev/constant_folding](https://github.com/tenstorrent/pytorch2.0_ttnn/tree/dev/constant_folding),  the changes also show here.

I have switched the base of this branch.
Thank you for pointing that out.